### PR TITLE
[Docs] Add image names and descriptions, how to build images, add/fix links

### PR DESCRIPTION
**Pull request by jillnogold**
[ML-4368](https://jira.iguazeng.com/browse/ML-4368)

### CODE REVIEW

File: docs/runtimes/images.md [hunk 1]
```diff
@@ -5,16 +5,60 @@ Every release of MLRun includes several images for different usages. The build a
...
+You can also build only a specific image, for example, `make api` (builds only the api image)
+Or a set of images, for example, `make mlrun jupyter base` 
+
+The possible commands are:
+- `mlrun`
+- `mlrun-gpu`
+- `api`
+- `jupyter`
+- `base`
+- `models`
```
Comment (liranbg) [inline docs/runtimes/images.md:58]: remove

**Review by liranbg — CHANGES_REQUESTED** [review]
mlrun/mlrun - An MLRun image includes preinstalled OpenMPI. Useful as a base image for simple jobs.

File: docs/runtimes/images.md (hunk 1 above)
Comment (jillnogold) [reply docs/runtimes/images.md:58]: @liranbg That means that this:
You can also build only a specific image, for example, `make api` (builds only the api image)
Or a set of images, for example, `make mlrun jupyter base` 
can only be replaced with
You can also build only a specific image, for example, `make mlrun` (builds only the api image)
Or a set of images, for example, `make mlrun mlrun-gpu`
But is there a use case for the 2nd example?

Comment (jillnogold) [reply docs/runtimes/images.md]: implemented without 2nd line

File: docs/runtimes/images.md [hunk 2]
```diff
@@ -5,16 +5,55 @@ Every release of MLRun includes several images for different usages. The build a
...
 
-See [README](https://github.com/mlrun/mlrun/blob/development/dockerfiles/README.md).
+Every release of MLRun includes several images for different usages. All images are published to 
+[dockerhub](https://hub.docker.com/u/mlrun) and [quay.iohttps://quay.io/organization/mlrun](https://quay.io/organization/mlrun]).
+
+The images are:
+
+`mlrun/mlrun`: An MLRun image includes preinstalled OpenMPI. Useful as a base image for simple jobs.
+`mlrun/mlrun-gpu`: The same as `mlrun/mlrun` but for GPUs, including Open MPI. 
+`mlrun/ml-base`: The image for file acquisition, compression, dask jobs, simple training jobs and other utilities. Like `lrun/mlrun` with the addition of Miniconda and other python packages.
```
Comment (liranbg) [inline docs/runtimes/images.md:25]:
```suggestion
`mlrun/ml-base`: The image for file acquisition, compression, dask jobs, simple training jobs and other utilities. Like `mlrun/mlrun` with the addition of Miniconda and other python packages.
```

File: docs/runtimes/images.md [hunk 3]
```diff
@@ -5,16 +5,55 @@ Every release of MLRun includes several images for different usages. The build a
...
+- MLRUN_DOCKER_REPO is the docker repository (defaults to mlrun)
+- MLRUN_DOCKER_REGISTRY is the docker registry (e.g. quay.io/, gcr.io/, defaults to empty (docker hub))
+
+For example, running `MLRUN_VERSION=x.y.z make docker-images` generates these images:
+- mlrun/mlrun-api:x.y.z
+- mlrun/mlrun:x.y.z
+- mlrun/mlrun-gpu:x.y.z
+- mlrun/jupyter:x.y.z
+- mlrun/ml-base:x.y.z
+- mlrun/ml-models:x.y.z
```
Comment (liranbg) [inline docs/runtimes/images.md:47]:
```suggestion
```

**Review by liranbg — COMMENTED** [review]
minor

**Pull request closed without merging**